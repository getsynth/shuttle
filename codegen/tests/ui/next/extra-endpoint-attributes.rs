shuttle_codegen::app! {
    #[shuttle_codegen::endpoint(method = get, route = "/hello")]
    #[shuttle_codegen::endpoint(method = post, route = "/hello")]
    async fn hello() -> &'static str {
        "Hello, World!"
    }
}
