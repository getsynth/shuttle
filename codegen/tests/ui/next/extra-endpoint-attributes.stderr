error: extra endpoint attribute

         = help: There should only be one endpoint annotation per handler function.

 --> tests/ui/next/extra-endpoint-attributes.rs:2:5
  |
2 | /     #[shuttle_codegen::endpoint(method = get, route = "/hello")]
3 | |     #[shuttle_codegen::endpoint(method = post, route = "/hello")]
4 | |     async fn hello() -> &'static str {
5 | |         "Hello, World!"
6 | |     }
  | |_____^

error[E0601]: `main` function not found in crate `$CRATE`
 --> tests/ui/next/extra-endpoint-attributes.rs:7:2
  |
7 | }
  |  ^ consider adding a `main` function to `$DIR/tests/ui/next/extra-endpoint-attributes.rs`
